### PR TITLE
Tweaks in general for both frontend and backend

### PR DESCRIPTION
**Pull request by hammerfunk**
Fixed some admin styling
Added some placeholder admin stuff for previous winnings
Restyled the add accessors
Fixed the admin each user controller for updating and creating
Fixed frontend styling
Fixed frontend for accessibility

### CODE REVIEW

File: app/assets/stylesheets/admin/page-applications.scss [hunk 1]
```diff
@@ -253,9 +253,45 @@
...
-  width: 5.5em;
-  font-weight: normal;
+.section-applicant-users {
+  li {
+    position: relative;
+    clear: both;
+  }
+
+  label {
+    float: left;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Properties should be ordered width, margin-bottom, float, font-weight

File: app/views/admin/form_answers/_assessors_section.html.slim [hunk 2]
```diff
@@ -1,23 +1,56 @@
...
+
+    li.form-group
+      label
         span.icon-role 1
-        label Primary:
-        = resource.assessors.primary.try(:full_name)
-    li
-      = link_to "#", class: "ellipsis"
+        ' Primary:
+      - if current_subject.try(:lead?, resource) || current_subject.is_a?(Admin)
```
Comment (dreamfall) [inline app/views/admin/form_answers/_assessors_section.html.slim]: I dont think we need `try` here.

**Comment by jaleszek** [comment]
going to check this specs

**Comment by jaleszek** [comment]
looks good :+1:

**Pull request merged**